docs/src/main/asciidoc/hibernate-orm.adoc

```diff
@@ -70,7 +70,7 @@ quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/hibernate_db
 
 # drop and create the database at startup (use `update` to only update the schema)
 quarkus.hibernate-orm.database.generation=drop-and-create
---
+----
 
 Note that these configuration properties are not the same ones as in your typical Hibernate ORM configuration file: these drive Quarkus configuration properties,
```

Comment:
'_...file: these drive Quarkus..._' -> this is a bit confusing. Do you actually need to say that 'they are not the same...'? Isn't enough to say 'Note that these configuration properties drive the Quarkus configuration properties, which often will map to Hibernate configuration properties, but could have different names and don't necessarily map 1:1 to each other.

Also, it's to clear what 'drives' mean.

Even better if you could name 'these' properties - 'Note that the datasource configuration properties drive (supply, produce, are the source for ??) the Quarkus configuration properties, which often will map to Hibernate configuration properties but they could have different names and don't necessarily map 1:1 to each other.

Reply:
Thanks @andreamussap ! I applied your other suggestions. For this one, I simplified things a bit by removing the `drive` part which was indeed confusing.